Lesson_7/Tusk.py

```diff
@@ -0,0 +1,71 @@
+phone_list = {}
+print('Phone list program is started.')
+
+commands = ('You can use next commands:'
+            '\n  - stats: If you want to see the quantity of records'
+            '\n  - add: if you want to add a new record'
+            '\n  - delete <name>: if you want to delete a record with name in <>'
+            '\n  - list: to see the list of all records'
+            '\n  - show <name>: to see the information about the name in <>'
+            '\n  - commands: if you want to see tha command list again'
+            '\n  - exit: if you want to exit the program')
+print(commands)
+
+while True:
+    command = input('Please enter a command: ').lower()
+    if command == 'stats':
+        if len(phone_list) == 0:
+            print('There is no records for now')
+        else:
+            print(len(phone_list))
+
+    elif command == 'add':
+        name = input('Please enter a name: ')
+        if name in phone_list:
+            print('This name is already here')
+            continue
+        phone_nr = input('Please enter a number: ')
+        if phone_nr.isdigit():
+            phone_list[name] = phone_nr
+            print(f'The name {name} added')
+        else:
+            print('Only number required, please try again')
+
+    elif 'delete ' in command:
+        del_name = command.split()[1]
+        if del_name[0] == '<' and del_name[-1] == '>':
```

Comment:
I didn't get this part: why do you need these symbols around the name?

Reply:
The command in tusk was "delete <name>". I thought <name> is important part of the tusk.

Reply:
Also that was interesting to practice how to use some text inside of a string.